R/get_merge_call.R

```diff
@@ -130,7 +130,7 @@ get_merge_key_grid <- function(selector_list, join_keys = teal.data::join_keys()
           get_merge_key_pair(
             selector_from,
             selector_to,
-            join_keys$get(selector_from$dataname, selector_to$dataname)
+            join_keys[selector_from$dataname, selector_to$dataname]
```

Comment:
If you use `[[` than this package could be independent of `JoinKeys` and work with a simple list (which is true in case of JoinKeys). In the examples we could use `join_keys()` constructor, but then `teal.data` would be only in `Suggests:`
```suggestion
            join_keys[[selector_from$dataname]][[selector_to$dataname]]
```

Reply:
As `teal.data` is already in `Imports` and we use it for default `join_keys` args too I thought we will never make it `Suggests` but I like reducing external package dependency wherever possible 👍🏽